src/static/webcomponents/topdowncompass/topdowncompass.html

```diff
@@ -0,0 +1,52 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+
+<polymer-element name="rov-topdowncompass">
+  <template>
+    <link rel="stylesheet" href="/css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/topdowncompass.css">
+
+    <style type="text/css">
+    </style>
+
+    <div id="compass" class="topdowncompass pull-left">
+      <div id="needle" class="compassNeedle"></div>
+    </div>
+
+  </template>
+  <script>
+    (function() {
+
+      Polymer({
+        ready: function () {
+          var compass = this;
+          window.cockpit.rov.on('plugin.navigationData.data', function(data) {
```

Comment:
what exactly do you mean by ** Having the child components have explicit references to the parents** [...]?

If you mean the ```window.cockpit.rov``` then you I have no idea right now on how to solve elegantly.